app/src/main/java/com/pvp/app/ui/screen/settings/SettingsScreen.kt

```diff
@@ -173,6 +175,118 @@ private fun SettingHydrationNotificationToggle(
     )
 }
 
+@Composable
+private fun SettingApplicationTheme(
+    model: SettingsViewModel = hiltViewModel()
+) {
+    val themeValue by model
+        .get(Setting.ApplicationTheme)
+        .collectAsStateWithLifecycle()
+
+    val state = rememberPickerState(initialValue = themeValue)
+
+    val valueText = when (themeValue) {
+        1 -> "Dark"
+        2 -> "Light"
+        3 -> "Dynamic"
```

Comment:
Dynamic can also be dark or light. You should allow choosing those too. But it becomes repetitive then. So different solution could be made: a `switch` for dynamic and a `checkbox` for dark, light themes. We should also have "auto", which checks device's theme and uses it, in case we can't find device's theme (old device), we would fallback into light theme.